### ISSUE TITLE
Create a cookbook for rsnapshot backups

### ISSUE THREAD

**Issue by carlalexander** [carlalexander/debops-wordpress#53]
I used `debops.rsnapshot` recently. It's easy to setup and works out of the box. It's probably worth creating a cookbook to explain how to set it up, how to do restores, etc.

**Comment by tobeycodes** [comment]
Can we do daily incremental and weekly full backup with rsnapshot? And multiple servers backup to a single backup server?

Very excited for this. DO image backups is all I've got at the moment

**Comment by carlalexander** [comment]
Yep, to all of these. It does hourly incremental backups by default if I remember correctly.

**Comment by matthew6688** [comment]
Wait for the instructions, which hosting support snapshot?

**Comment by carlalexander** [comment]
Digital Ocean does backups if you turn them on. To support `debops.rsnapshot`, you'll need a backup server. But you could use that backup server for multiple servers.
